docs/vocascan-server/traefik/traefik.md

```diff
@@ -0,0 +1,149 @@
+# Installation with Traefik
+
+This guide will get you through the installation of your own vocascan-server paired with Traefik as a reverse proxy and
+Let's Encrypt.
+
+1. First you have to install Docker and Docker-Compose. To do this have a look at point 1 and 2 our
+   [guide](vocascan-server/docker/docker) in the general vocascan-server installation section.
+
+2. After that create a secure password hash for the Traefik monitoring website with
+   `htpasswd`
+
+   ```
+   docker run --entrypoint htpasswd httpd:2 -Bbn admin your_password
```

Comment:
the `--rm` flag is needed to remove the container after exit so you dont have a stopped container.